tests/fixtures/inversify-dynamic-container/ioc.ts

```diff
@@ -0,0 +1,14 @@
+import e = require('express');
+import { Container } from 'inversify';
+import { ManagedController } from './managedController';
+import { ManagedService } from './managedService';
+
+const iocContainer = function (request: e.Request): Container {
```

Comment:
Use the new import

Reply:
Will do! Do you think this is needed on all the tests?

Reply:
If you use it for the ioc modules in /inversify and /inversify-cpg while you're at it, I wouldn't complain :smile:

Reply:
I've just realised I can't type these tests because the containers use `.bind` and `.load` which aren't on the interface.